src/riak_cs_user.erl

```diff
@@ -30,6 +30,7 @@
          is_admin/1,
          get_user/2,
          get_user_by_index/3,
+         from_riakc_obj/2,
```

Comment:
Ah. This was remain of my effort to try consistency updating user object. Fruitless effort without strong consistency.